src/components/TeamMemberTasks/TaskAndProgressBar.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { getProgressColor, getProgressValue } from '../../utils/effortColors';
+import { Progress } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBell } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+
+const TaskAndProgressBar = ({ task, user }) => {
+  return (
+    <div className="team-member-task">
+      <p className="team-member-task-name">
+        <Link to={task.projectId ? `/wbs/tasks/${task._id}` : '/'}>
+          <span>{`${task.num} ${task.taskName}`} </span>
+        </Link>
+        {task.taskNotifications.length > 0 && (
+          <FontAwesomeIcon
+            className="team-member-tasks-bell"
+            icon={faBell}
+            onClick={() => {
+              handleOpenTaskNotificationModal(user.personId, task, task.taskNotifications);
```

Comment:
The red Bell icon in tasks seems not working, probably because `handleOpenTaskNotificationModal` here is not defined. I think the simplest way here is to make it a parameter of `TaskAndProgressBar`

Reply:
Thanks for the catch. I guess maybe I delete that function accidentally when I was restructuring the code...